<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cpuregs</class>
 <widget class="QWidget" name="cpuregs">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CPU Registers</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QMenuBar" name="menubar">
     <widget class="QMenu" name="menu_file">
      <property name="title">
       <string>&amp;File</string>
      </property>
      <addaction name="action_file_save"/>
     </widget>
     <widget class="QMenu" name="menu_options">
      <property name="title">
       <string>&amp;Options</string>
      </property>
      <addaction name="action_options_autorefresh"/>
      <addaction name="action_options_textview"/>
     </widget>
     <addaction name="menu_file"/>
     <addaction name="menu_options"/>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="stack">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page_regs"/>
     <widget class="QWidget" name="page_text">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QTextEdit" name="out_cpuregs">
         <property name="font">
          <font>
           <family>Monospace</family>
          </font>
         </property>
         <property name="lineWrapMode">
          <enum>QTextEdit::NoWrap</enum>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
  <action name="action_file_save">
   <property name="text">
    <string>&amp;Save</string>
   </property>
  </action>
  <action name="action_options_autorefresh">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Auto Refresh</string>
   </property>
  </action>
  <action name="action_options_textview">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Text View</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
